core/common/src/PlanarClipMask.ts

```diff
@@ -47,71 +41,90 @@ export enum PlanarClipMaskPriority {
   /** A reality model with a bounded range. */
   RealityModel = 0,
   /** A design model stored in the [IModelDb]($backend). */
-  BIM = 2048,
-  /** The maximum supported priority. */
-  Maximum = 4096,
```

Comment:
Maximum intentionally removed?

Reply:
Yes, as discussed with Ray - nothing enforces/checks it.